feature-libs/product-configurator/rulebased/components/price/configurator-price.component.html

```diff
@@ -17,3 +17,8 @@
     <div class="cx-price-total">{{ priceTotal }}</div>
   </ng-container>
 </ng-container>
+<ng-container *ngIf="displaySelected()">
```

Comment:
I would suggest to define an else case and add a template for it, like

> <ng-container *ngIf="displayFormula(); else selectedCase">
>   <ng-container *ngIf="displayPriceOnly()">
>     <div
>       [ngClass]="styleClass"
>       [attr.aria-label]="'configurator.a11y.valueSurcharge' | cxTranslate"
>     >
>       {{ price }}
>     </div>
>   </ng-container>
>   <ng-container *ngIf="displayQuantityAndPrice()">
>     <div
>       class="cx-quantity-price"
>       [attr.aria-label]="'configurator.a11y.valueSurcharge' | cxTranslate"
>     >
>       {{ quantityWithPrice(formula?.quantity | cxNumeric) }}
>     </div>
>     <div class="cx-price-total">{{ priceTotal }}</div>
>   </ng-container>
> </ng-container>
> 
> <ng-template #selectedCase>
>   <ng-container *ngIf="isPriceLightedUp()">
>     <div [ngClass]="styleClass">
>       {{ 'configurator.price.selectedItem' | cxTranslate }}
>     </div>
>   </ng-container>
> </ng-template>

Then `displaySelected` method could be completely removed

Reply:
Plz, check with a11y whether the `Selected` will be read by the screen reader. I strongly believe not because the `attr.aria--label` is not added. It should be like:
 
> <div [ngClass]="styleClass"
>          [attr.aria-label]="'configurator.price.selectedItem' | cxTranslate">
>       {{ 'configurator.price.selectedItem' | cxTranslate }}
>     </div>